pandas/compat/_optional.py

```diff
@@ -36,7 +36,7 @@
     "pymysql": "1.0.2",
     "pyarrow": "7.0.0",
     "pyreadstat": "1.1.5",
-    "pytest": "7.0.0",
+    "pytest": "7.3.2",
```

Comment:
@kathleenhang I'd have forgotten this one, but your script caught it, thanks 🙌

Reply:
Thanks for the ping! It's really cool to see how it's helpful.